util/nrtClean.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source suite
+# of simulation tools for power systems.
+
+
+import os
+import sys
+import shutil
+import re
+
+
+LOAD_INCREASE_FINAL_STATE_REGEX = re.compile('^loadIncreaseFinalState-[0-9]*.iidm$')
+
+if not os.getenv("DYNAWO_ALGORITHMS_NRT_DIR"):
+    print("environment variable DYNAWO_ALGORITHMS_NRT_DIR needs to be defined")
+    sys.exit(1)
+
+
+def find_nrt_folders():
+    zip_input_nrt_folders = list()
+    fic_multiple_nrt_folders = list()
+
+    nrt_dir = os.path.join(os.getenv("DYNAWO_ALGORITHMS_NRT_DIR"), "data", "IEEE14")
```

Comment:
Another folder than "IEEE14" could be created, we need to iterate inside all the folders in "data"